python/hsfs/core/statistics_api.py

```diff
@@ -78,6 +78,15 @@ def get_last(self, metadata_instance, for_transformation, training_dataset_versi
             _client._send_request("GET", path_params, query_params, headers=headers)
         )
 
+    def get_by_feature_monitoring_window_config(self, monitoring_window_config):
```

Comment:
It is fine for now but I think the logic of transforming a monitoring window config into search parameters for a specific stat should be handled in an engine class, not directly in the API